Documentation/GETTING_STARTED.md

```diff
@@ -13,7 +13,10 @@
 
 * It can be a VM, physical server or container.
 * It requires CentOS 7.x or RHEL 7.x with basic packages.
-* We recommend that you dedicate a machine (VM) for it.  
+* We recommend that you dedicate a machine that has following requirements:
+  * Use a dedicated server on VM
+  * Minimum of 2 CPUs, 4 GBs memory, and 40 GB disk
+  * A read/write NFS mount for accessing the Nuage software images  
 
 2.1 Clone the master branch of the repo onto the **MetroAEe Host**. Read [Setup](SETUP.md) for details.  
```

Comment:
I know you didn't put it there, but could you fix the "MetroAEe" to remove the extra "e"?

Reply:
sure done